src/main/java/bio/terra/pdao/bigquery/BigQueryConfiguration.java

```diff
@@ -0,0 +1,41 @@
+package bio.terra.pdao.bigquery;
+
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * The theory of operation here is that each implementation of a pdao will specify a configuration.
+ * That way, implementation specifics can be separated from the interface. We'll see if it works out that way.
+ */
+@Configuration
+@Profile("bigquery")
+public class BigQueryConfiguration {
+
+    private static String rowIdColumnName = "datarepo_row_id";
+    private static String rowIdColumnDatatype = "STRING"; // TODO: datatype enums??
```

Comment:
I added a ticket along with a doc for discussing "our" datatypes -> BQ datatypes.

https://broadinstitute.atlassian.net/jira/software/projects/DR/boards/517/backlog?selectedIssue=DR-118

Reply:
I had already added DR-112. I moved the contents of your description in DR-118 into DR-112, marked DR-118 as a duplicate and Done.